build/packages/rpm/server/SPECS/grafana.spec

```diff
@@ -1,9 +1,9 @@
 %global debug_package   %{nil}
-%global commit          7eb13cd5e16e25e8392845a719ed8a3cc634b3f7
+%global commit          c44b7a022e284a783df28c9f53acb50aab067f7d
```

Comment:
Commit hash should be the latest one `after` the main grafana PR is merged.

(Merge grafana PR - when approved etc. - use the commit hash here)